recipes/xcube-geodb/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: xcube_geodb
+  version: {{ version }}
+
+source:
+  url: https://github.com/dcs4cop/xcube-geodb/archive/0.1.0.dev0.tar.gz
+  sha256: ad67c4f6a53ea482b2ce738a3cd77df5d1d8679bcc50291756d6e7e7ae9fd226
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - geodb = xcube_geodb.cli.main:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - pip
+    - xcube >=0.3.0
+    - geopandas >=0.5
+    - matplotlib
```

Comment:
Can this be matplotlib-base or does it need the Qt backend?

Reply:
matplotlib-base should be fine